script.js

```diff
@@ -1,4 +1,147 @@
+// global variable
+//user name input
+var currentGameMode = `awaiting user name`;
+var userName1 = "";
+var userName2 = "";
+var noOfRounds = 0;
+
+//dice roll
+//player 1
+var diceRoll1 = [];
+var diceRoll2 = [];
+var player1Num = "";
+
+//player 2
+var diceRoll3 = [];
+var diceRoll4 = [];
+var player2Num = "";
+
+//scores
+var player1score = "";
+var player2score = "";
+
+//roll dice function
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+  currentGameMode == "awaiting user name";
+  if (currentGameMode == "awaiting user name") {
+    userName1 = input;
+    myOutputValue =
+      `Welcome ` + userName1 + `! <br><br> Player 2, please input your name.`;
+    currentGameMode = "awaiting user name 2";
+  } else if (currentGameMode == "awaiting user name 2") {
+    userName2 = input;
+    myOutputValue =
+      `Welcome ` +
+      userName2 +
+      `! <br><br>` +
+      userName1 +
+      ` , please press submit to roll the dice`;
+    console.log("myOutputValue2");
+    console.log(myOutputValue);
```

Comment:
Good labeling and use of console logs!
You could also combine them into one line `console.log("myOutputValue2", myOutputValue)` (just for shorter, concise code!)